src/test/scala/org/ergoplatform/appkit/ergotool/ErgoToolSpec.scala

```diff
@@ -267,5 +267,33 @@ class ErgoToolSpec extends PropSpec with Matchers with ScalaCheckDrivenPropertyC
     println(res)
     res should include ("\"transactionId\": \"f3e37a37b561a34bf91f37c9f9fbed1eb42a4d9bb364f869d42bcde22f5d8229\",")
   }
+
+  property("AssetsAtomicExchange list command") {
+    val data = MockData(
+      Seq(
+        loadNodeResponse("response_Box_AAE_seller_contract.json"),
+        loadNodeResponse("response_Box1.json"),
+        loadNodeResponse("response_Box2.json"),
+        loadNodeResponse("response_Box3.json"),
+        loadNodeResponse("response_Box_AAE_buyer_contract.json"),
+        loadNodeResponse("response_Box1.json"),
+        loadNodeResponse("response_Box2.json"),
+        loadNodeResponse("response_Box3.json"),
+        ),
+      Seq(
+        loadExplorerResponse("response_boxesByAddressUnspent.json"),
+        loadExplorerResponse("response_boxesByAddressUnspent.json")
+      )
+    )
+
+    val res = runCommand("AssetAtomicExchangeList",
```

Comment:
short unambiguous names with prefix namespace

Reply:
I presume you are talking about https://github.com/aslesarenko/ergo-tool/issues/12.

Reply:
Done